helper-upload-assets-to-s3

```diff
@@ -0,0 +1,68 @@
+#!/usr/bin/env bash
+#
+# HELPER PURPOSE
+#
+# Uploads the compressed assets generated by apps using Page kit to S3
+#
+# This helper will be executed in place of the current deploy-hashed-assets
+# command in n-heroku-tools for all apps that are using Page Kit.
+#
+# It uploads JavaScipt, CSS, sourcemaps and compressed files to the EU and US
+# regions, making the assets available to our CDN layer.
+#
+
+# Set error handling
+set -x -eu -o pipefail
+
+# HELPER COMMANDS
+
+
+DESTINATION_BUCKET_EU="ft-next-hashed-assets-prod"
+DESTINATION_BUCKET_US="ft-next-hashed-assets-prod-us"
+DESTINATION_FOLDER="hashed-assets/page-kit"
+SOURCE_FOLDER="public"
```

Comment:
I think `DESTINATION_FOLDER` and `SOURCE_FOLDER` should be args but these could be the default values, e.g.:

```bash
DESTINATION_FOLDER=${2:-"hashed-assets/page-kit"}
SOURCE_FOLDER=${1:-"public"}
```